nevergrad/benchmark/experiments.py

```diff
@@ -43,6 +43,28 @@
 # fmt: off
 
 
+class _Constraint:
+
+    def __init__(self, name: str, as_bool: bool) -> None:
+         self.name = name
+         self.as_bool = as_bool
+
+    def __call__(self, data: np.ndarray) -> tp.Union[bool, float]:
+        if not isinstance(data, np.ndarray):
+            raise ValueError(f"Unexpected inputs as np.ndarray, got {data}")
+        if self.name == "positive_sum":
+            value = float(np.sum(data))
+        elif self.name == "positive_diff":
+            value = float(np.sum(data[::2]) - np.sum(data[1::2]))
+        elif self.name == "positive_second_diff":
+            value = float(2 * np.sum(data[1::2]) - 3 * np.sum(data[::2]))
```

Comment:
```suggestion
        if self.name == "sum":
            value = float(np.sum(data))
        elif self.name == "diff":
            value = float(np.sum(data[::2]) - np.sum(data[1::2]))
        elif self.name == "second_diff":
            value = float(2 * np.sum(data[1::2]) - 3 * np.sum(data[::2]))
```
it's definitely not positive so the naming is confusing

Reply:
The constraint is checking if this is positive, hence the naming -- but ok for changing.